sdk/communication/communication-chat/src/signaling/signalingClient.ts

```diff
@@ -19,9 +19,10 @@ export const getSignalingClient = (
   if (typeof navigator !== "undefined" && navigator.product === "ReactNative") {
     // In React Native
     return new CommunicationSignalingClient(credential, logger, {
-      environment: options?.environment ?? undefined,
       resourceEndpoint: options?.resourceEndpoint ?? undefined,
       gatewayApiVersion: options?.gatewayApiVersion ?? undefined,
+      additionalPolicies: options?.additionalPolicies ?? undefined,
```

Comment:
In chat client, the user agent options/retry policies need to be passed into the signaling client options.